packages/components/src/angle-picker-control/index.tsx

```diff
@@ -40,16 +50,20 @@ export default function AnglePickerControl( {
 		);
 	}
 
-	const handleOnNumberChange = ( unprocessedValue ) => {
+	const handleOnNumberChange = ( unprocessedValue: string | undefined ) => {
 		const inputValue =
-			unprocessedValue !== '' ? parseInt( unprocessedValue, 10 ) : 0;
+			unprocessedValue !== undefined && unprocessedValue !== ''
```

Comment:
Runtime change: added  an `undefined` type guard to fix a TypeScript error (previously the component would attempt to `parseInt( undefined )`, now it fallbacks to `0`